src/battle-dex.ts

```diff
@@ -755,6 +755,17 @@ const Dex = new class implements ModdedDex {
 		let sanitizedType = type.replace(/\?/g, '%3f');
 		return '<img src="' + Dex.resourcePrefix + 'sprites/types/' + sanitizedType + '.png" alt="' + type + '" height="14" width="32"' + (b ? ' class="b"' : '') + ' />';
 	}
+
+	getPokeballs() {
```

Comment:
I think we should probably cache these instead of looping through 300+ items each time?

Reply:
Possibly? Stick them in `Dex.pokeballs`?